cli/sql.go

```diff
@@ -57,6 +57,28 @@ Open a sql shell running against the cockroach database at --addr.
 	Run: runTerm,
 }
 
+type sqlValue struct {
+	value string
+}
+
+func (s *sqlValue) Scan(value interface{}) error {
+	switch v := value.(type) {
+	case nil:
+		s.value = "NULL"
+	case string:
+		s.value = v
+	case []byte:
+		s.value = string(v)
+	case int8, int16, int32, int64:
+		s.value = fmt.Sprintf("%d", v)
+	case float32, float64:
+		s.value = fmt.Sprintf("%f", v)
+	default:
+		s.value = fmt.Sprintf("%v", value)
```

Comment:
I think you have too many types in this type switch. `sql.Scanner` guarantees the incoming `value` has a more limited set of types: http://golang.org/pkg/database/sql/#Scanner.